windows/RNFS/RNFS.def

```diff
@@ -0,0 +1,3 @@
+﻿EXPORTS
+DllCanUnloadNow = WINRT_CanUnloadNow                    PRIVATE
```

Comment:
I was testing this PR in our code via a patch and was getting weird error around this file when it wasn't using crlf (it appears to be currently lf)  Visual Studio 2019 compiler errored out on it.

Consider updating to crlf?